stairway/src/main/java/bio/terra/stairway/impl/StairwayThreadPool.java

```diff
@@ -67,16 +48,4 @@ private void initializeFlightMdc(
     MdcUtils.addFlightContextToMdc(flightContext);
     MdcUtils.removeStepContextFromMdc(flightContext);
   }
-
-  protected void beforeExecute(Thread t, Runnable r) {
```

Comment:
I see these metrics were removed -- are these things we get "for free" with the switch to the Spring `ThreadPoolTaskExecutor`?

Reply:
Happily, yes!  And they're more valuable to us as a metric rather than a debug log.  They're available as the metric `executor_active_threads`.

```
(base) okotsopo@wm111-e35 jade-data-repo % curl http://localhost:8080/actuator/prometheus | grep 'stairwayExecutor' | grep 'executor_active_threads'
…
executor_active_threads{name="stairwayExecutor",} 1.0
```